### ISSUE TITLE
A11y_AzureDataStudio_Deploymenttemplateconfiguration_Keyboard:Not able to select the "Aks -dev-test tab" with the help of keyboard.

### ISSUE THREAD

**Issue by V-sheman** [microsoft/azuredatastudio#7944]
GitHubTags: #A11y_AzureDataStudio;#A11yMAS; #A11yTCS;#SQL Azure Data Studio;#WCAG 2.1.1#Keyboard;DesktopApp;#Voice Over#Mac
**Environment Details:**Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: MacOS Mojave 10.14.6
**Repro Steps:**
1.Install /Launch the application Azure data studio.
2.Navigate to connections and click on more action button.
3.Select “Deploy SQL Server big data cluster” from more action drop down.
4.Click on “Deploy SQL Server big data cluster”.
5. "Select the deployment option" screen gets open.
6.check the I accept check box and click on select button.
7. Deployment template configuration screen gets open.
8. Try to select the Aks -dev-test tab" by pressing enter or space bar.
**Actual:**
Not able to select the Aks -dev-test tab" with the help of keyboard.
**Expected:**
We should be able to select the data with the help of keyboard.
**User Impact:**
keyboard user will  navigate through the screen smoothly.

**Attachment for Reference:**
[7944_A11y_AzuredataStudio_Deploymenttemplateconfiguration_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3761563/7944_A11y_AzuredataStudio_Deploymenttemplateconfiguration_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by vinodbk197** [comment]
Verified this issue on:
Azure Data Studio
Version: 1.15.0-insider (user setup)

The issue is no longer to reproduce, Hence closing this issue.
[7944_Closed.pptx](https://github.com/microsoft/azuredatastudio/files/4138888/7944_Closed.pptx)

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_Mac;

**Comment by sspatil123** [comment]
#A11ySev2;
